src/components/PostInfo/PostInfo.tsx

```diff
@@ -1,5 +1,42 @@
 import React from 'react';
+import { Post } from '../../types/Post';
+import { CommentList } from '../CommentList';
+import { UserInfo } from '../UserInfo';
+import './PostInfo.scss';
 
-export const PostInfo: React.FC = () => (
-  <>Put the post here</>
-);
+type Props = {
+  post: Post;
+};
+
+export const PostInfo: React.FC<Props> = ({ post }) => {
+  const {
+    title,
+    body,
+    user,
+    comments,
+  } = post;
+
+  return (
+    <div className="PostInfo">
+      <div className="PostInfo__header">
+        <h3 className="PostInfo__title">{ title }</h3>
+
+        <p>
+          {' Posted by  '}
+
+          {user
+            ? <UserInfo user={user} />
+            : ''}
+        </p>
```

Comment:
```suggestion
{user && <UserInfo user={user} />}
```

1. This paragraph belongs to UserInfo (if no user - no paragraph with text Posted by...)
2. Same condition can be achieved with AND operator.